DonoGear/app/src/main/java/com/example/donogear/actionpages/MainActivity.java

```diff
@@ -197,6 +210,52 @@ private void readData() {
                 Log.e("Error", e.toString());
             }
         });
+
+        ParseQuery<ParseObject> announcementQuery = ParseQuery.getQuery(ANNOUNCEMENTS);
```

Comment:
I'd suggest adding a comment above this query statement to explain the purpose of this query